pydevd_file_utils.py

```diff
@@ -73,10 +73,31 @@
 
 normcase = os_normcase # May be rebound on set_ide_os
 
+CTYPES_AVAILABLE = True
+try:
+    import ctypes
+except ImportError:
+    CTYPES_AVAILABLE = False
+
+
+def convert_to_long_pathname(filename):
+    if CTYPES_AVAILABLE:
+        buf = ctypes.create_unicode_buffer(260)
+        GetLongPathName = ctypes.windll.kernel32.GetLongPathNameW
+        if IS_PY2:
+            filename = unicode(filename)
```

Comment:
We need to pass unicode string to the function `GetLongPathNameW`, that's why we convert it.